src/electron/electron/url.cljs

```diff
@@ -64,6 +64,12 @@
       (= "auth-callback" url-host)
       (send-to-renderer win "loginCallback" (.get (.-searchParams parsed-url) "code"))
 
+      (= "quick-capture" url-host)
```

Comment:
Do we want this url to eventually add to another graph or specific block? Is the url intended for any third party app, including commandline tools? This could be a useful url for the `logseq` cli to use. Asking about this to make sure the name reflects the usage. Other names could be `graph/add` or `add-to-graph`

Reply:
Also be aware that in multi-window case, the quick capture is added to the "main window" but not the focused window

Reply:
It might be useful to quickly capture into a specific graph or page (not sure about a specific block).
> Is the url intended for any third party app, including commandline tools? This could be a useful url for the logseq cli to use.

It'll be great if this could be used by CLI and other apps.

Reply:
- `quick-capture`
- `add-to-graph`
- `share`
- `import`

Any new ideas? This URL will not be used to add a standalone graph, but only assets, blocks, or pages.